### ISSUE TITLE
Source of the mystery 1.class

### ISSUE THREAD

**Issue by Chocohead** [FabricMC/intermediary#5]
The remapped game jar has had a class remapped to `1.class` ever since 19w12b which was presumed to be an anonymous class at the time in Discord IIRC. Turns out it is not anonymous at all but is in fact `OptionButtonWidget`. Taking a look one step at a time:

In the 1.14 Yarn mappings `cwz` is mapped to produce `1.class`:
```
CLASS	cwz	net/minecraft/client/gui/widget/OptionButtonWidget	net/minecraft/1
FIELD	cwz	Lcvl;	a	field_18970	field_18970
```
As per the 1.14 Intermediaries on lines 23725-6
```
CLASS	cwz	net/minecraft/1
FIELD	cwz	Lcvl;	a	field_18970
```

Going back in time to track `field_18970`, the first reference to it was 19w12b (on lines 23327-8)
```
CLASS	cvp	net/minecraft/1
FIELD	cvp	Lcud;	a	field_18970
```
Conveniently the field is 10 before the field counter limit. Looking back to 19w12a however (on lines 23328-9) reveals it never should have been assigned a number that late:
```
CLASS	cvp	net/minecraft/class_349
FIELD	cvp	Lcud;	a	field_2141
```

Tracking `class_349` back reveals it has existed since the earliest Intermediary release of 18w43b (on lines 4453-5):
```
CLASS	ckw	net/minecraft/class_349
FIELD	ckw	Lcjm$a;	o	field_2141
METHOD	ckw	()Lcjm$a;	c	method_1899
```
Thus all along it was an incorrect remapping in 19w12b that both lost the old mapping, and produced the broken naming that has persisted ever since.

**Comment by modmuss50** [comment]
Seems like something that needs to be fixed for 1.14.1. I guess we can give it its old name back?

Or do we want to fix it now and hope no mods use it. Im leaning towards waiting for 1.14.1

**Comment by Chocohead** [comment]
It is public (if only client side), so it is probably safest to wait for 1.14.1 given it appears imminent. Doubt anyone is actually using it unless they've tinkered with one of the settings screens though.

**Comment by modmuss50** [comment]
Should be fixed in 1.14.1. Let me know if not
